tests/helpers.py

```diff
@@ -0,0 +1,33 @@
+"""Helper functions for our tests."""
+
+import os
+from contextlib import contextmanager
+
+
+@contextmanager
+def inside_dir(dirpath):
+    """
+    Execute code from inside the given directory.
+
+    :param dirpath: String, path of the directory the command is being run.
```

Comment:
Don't we still prefer Google/NumPy-style docstrings? https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#docstrings

Reply:
Yes! I must have started those from some copy/paste